src/cargo-fmt/main.rs

```diff
@@ -112,6 +116,18 @@ fn execute() -> i32 {
         }
     }
 
+    let mut cargo_metadata_additional_opts: Vec<String> = Vec::new();
```

Comment:
I elected to go this route to make it easier to add any additional options in the future